src/components/header/template.test.js

```diff
@@ -247,15 +247,6 @@ describe('header', () => {
 
   describe('SVG logo', () => {
     const $ = render('header', {});
-    const $svg = $('.govuk-header__logotype-crown');
-
-    it('sets focusable="false" so that IE does not treat it as an interactive element', () => {
```

Comment:
This test has been deleted but it looks like we still have the SVG with focusable="false", should we keep the test? Or delete the attribute?

Reply:
I replaced the SVG element with the one from the guidance on the govuk-frontend release notes. I am going to leave the test that tests for focusable="false" as this is what govuk recommends.